### DISCUSSION TITLE
How to install the specified version of openzeppelin using foundry

### DISCUSSION THREAD

**Discussion by IAm0xAxe** [Cyfrin/foundry-full-course-cu#347] [Q&A] [answered]
For example, I like to install openzeppelin v4.8.2 version, but it reports an error.

```
forge install Openzeppelin/openzeppelin-contracts@4.8.2 --no-commit

Installing openzeppelin-contracts in /root/Blockchain-Security-CTF/lib/openzeppelin-contracts (url: Some("https://github.com/Openzeppelin/openzeppelin-contracts"), tag: Some("4.8.2"))
fatal: not a git repository (or any of the parent directories): .git
Error:
git submodule exited with code 128
```

**Comment by Naman1729** [comment, marked as answer]
Hey @IAm0xAxe , you have to put v after "@"

So, you can write like below :
```bash
forge install Openzeppelin/openzeppelin-contracts@v4.8.2 --no-commit
```

**Comment by IAm0xAxe** [comment]
> Hey @IAm0xAxe , you have to put v after "@"
> 
> So, you can write like below :
> 
> ```shell
> forge install Openzeppelin/openzeppelin-contracts@v4.8.2 --no-commit
> ```
Thank you so much @Naman1729  you are right. I just solved the problem this way too: `forge install openzeppelin-contracts-06=OpenZeppelin/openzeppelin-contracts@v3.4.0 --no-commit`
